main.py

```
@@ -1,20 +1,310 @@
+        try:
+            target_uid = int(q.message.text.split("user ")[1].split(":")[0].strip())
+            closed = load(CLOSED_CHATS_DB)
+            if target_uid not in closed:
+                closed.append(target_uid); save(CLOSED_CHATS_DB, closed)
+            await q.message.edit_text(q.message.text + "\n\n🔒 (CHAT CLOSED)")
+            await context.bot.send_message(target_uid, "🚫 Admin closed this chat.")
+        except: await q.answer("Error")
+
+# ---------- HANDLERS ----------
+async def add_admin(update, context):
+    if update.effective_user.id != OWNER_ID: return
```

Ruff (0.14.10)

Multiple statements on one line (colon)

(E701)